libraries/botbuilder-config/package.json

```diff
@@ -22,8 +22,8 @@
     "msbot"
   ],
   "contributos": [
```

Comment:
Just noticed this type, maybe you can change it in this PR also? :-)
`contributos` -> `contributors`